### ISSUE TITLE
clipboard truncated when pasting long files into pythonista. 

### ISSUE THREAD

**Issue by joereddington** [omz/Pythonista-Issues#500]
I have a use case where I occasionally paste very long (log) files into *.md files in pythonista for parsing by a script.  

Today it became clear that althought iOS can fit the whole log into the clipboard (it pastes fine into Evernote) pythonista truncates the file before the end.  :(

**Comment by Phuket2** [comment]
@joereddington , do you have a char count? If you do some testing with slicing, you might see a magic number were the truncation occurs.

**Comment by Phuket2** [comment]
@joereddington , this seems like its like a memory thing on your device.  
I can do the below and get the correct result '0'.  But if I push the char count to say 100_000_000 * 3, then sometimes it crashes.  
I am on latest ios, latest Pythonista beta, 12.9' latest ipad highest specs.

```python
import clipboard

char_count = 100_000_000 
text = '1' * char_count
text += '0'
clipboard.set(text)
print(clipboard.get()[-1])
```python

**Comment by zrzka** [comment]
@joereddington thanks for the report, couple of questions ...

* What is the source application? (App where you select text and hit copy)
* Can you attach your file causing this issue here?

**Comment by joereddington** [comment]
Ah - fixed by upgrading to 3.2 (which I should have tried earlier) thank you for your responses.
